offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/components/offer/create/OfferOverviewView.groovy

```diff
@@ -43,7 +36,8 @@ class OfferOverviewView extends VerticalLayout{
     Button save
     CreateOfferController createOfferController
 
-    OfferOverviewView(CreateOfferViewModel viewModel, CreateOfferController controller, OfferResourcesService service){
+    OfferOverviewView(CreateOfferViewModel viewModel, CreateOfferController controller, ResourcesService<Offer> service){
+        //FIXME the service is unused
```

Comment:
didnt we say the service is unused? Do we have an issue for that? Or where do we tackle it? Just that it does not get forgotten